### ISSUE TITLE
ToC heading should start at level 1

### ISSUE THREAD

**Issue by llemeurfr** [readium/readium-desktop#569]
The table of contents shows the headings starting at a level 3. This will be confusing to the reader. It is properly nested, but I suggest that it start at level 1 and not be a child of the interface of the page. What about putting a heading first and then start proper nesting? (G.Kersher)

**Comment by panaC** [comment]
each entry in TOC is a link "a" order from list "ul" "li".

in the hands of Laurent

**Comment by danielweck** [comment]
The TOC "starts" at level 3 because of the nested `ul` lists (the top-level `ul` is the horizontal icon bar at the top of the window, then the icon group aligned to the right, then etc. etc.).

https://github.com/readium/readium-desktop/blob/cf8be49c29afea718fb3db4c78a4f4d485368065/src/renderer/components/reader/ReaderMenu.tsx#L105-L164

https://github.com/readium/readium-desktop/blob/cf8be49c29afea718fb3db4c78a4f4d485368065/src/renderer/components/reader/ReaderHeader.tsx#L83-L192

**Comment by llemeurfr** [comment]
ok I see that. What can we do to avoid  this effect? 

To be discussed:
- we cannot change the top `ul` (directly in the nav bar) because using nav/ul is standard practice.
- we can certainly change the second level of `ul` (the group of 4 buttons on the right side) but as it supports a flot=right we must be careful (replacing it by a div does not work).  
- we can certainly change the third level of `ul` (list of navigation tools, ToC, Illustrations, Bookmarks, Annotations) for a set of `div`, also with caution about the semantic structure.

**Comment by danielweck** [comment]
`aria-level`
https://www.w3.org/TR/wai-aria-1.1/#aria-level

**Comment by llemeurfr** [comment]
@danielweck so does it mean that using aria-level would provide the expected behaviour without having to modify the "ul" imbrication?

**Comment by danielweck** [comment]
In principle. We have to verify that the screen readers pick-up the overridden nesting levels in the accessibility tree.

**Comment by Scarsniik** [comment]
I tried to use the aria-level and the screen reader don't seems to react
Or maybe I didn't understood how to use it

**Comment by danielweck** [comment]
I tried this PR with VoiceOver on MacOS ... to no avail:

https://github.com/readium/readium-desktop/pull/670/files

...I am also fixing bugs in the tree view, I will file a separate set of issues.

**Comment by danielweck** [comment]
Related issue: https://github.com/readium/readium-desktop/issues/671

**Comment by danielweck** [comment]
Related issue: https://github.com/readium/readium-desktop/issues/672

**Comment by danielweck** [comment]
After discussing with Leo, this issue is now reassigned to myself, as I have made some progress in the aforementioned PR (see 3 comments above)

**Comment by danielweck** [comment]
The implemented solution is not idea, but pragmatic to work with various screen readers.
Follow-up issue: https://github.com/readium/readium-desktop/issues/679
